src/Makefile

```diff
@@ -56,8 +56,10 @@ $(builddir)/$(LIB_MAJOR_VERSION): $(MAIN_OBJS)
 ifneq ($(OS),WINNT)
 $(builddir)/$(LIB_FULL_VERSION): | $(builddir)/$(LIB_MAJOR_VERSION)
 	ln -sf "$(LIB_MAJOR_VERSION)" "$@"
+endif
 
 $(builddir)/libblastrampoline.$(SHLIB_EXT): | $(builddir)/$(LIB_MAJOR_VERSION)
```

Comment:
A non-phony rule which doesn't generate the target is quite dodgy. Looks to there's a logic flaw here.